tla/consistency/TraceMultiNodeReads.tla

```diff
@@ -81,13 +86,29 @@ IsRoTxResponseAction ==
     /\ RoTxResponseAction
     /\ Last(history').type = ToTxType[logline.type]
     /\ Last(history').tx = logline.tx
+    /\ Last(history').tx_id = logline.tx_id
 
 IsStatusInvalidResponseAction ==
     /\ IsEvent("StatusInvalidResponseAction")
     /\ StatusInvalidResponseAction
     /\ Last(history').type = ToTxType[logline.type]
     /\ Last(history').status = ToStatus[logline.status]
 
+IsNotEvent ==
+    l' = l
```

Comment:
Allowing `l` to stutter will make TLC accept any prefix of the log because `Len(JsonLog)` repetition of `InsertOtherTxnAction ` actions will satisfy `TLCGet("stats").diameter >= Len(JsonLog)`.